packages/odyssey-react-mui/src/themes/odyssey/typography.types.ts

```diff
@@ -25,14 +25,13 @@ declare module "@mui/material/styles" {
 
 declare module "@mui/material/Typography" {
   interface TypographyPropsVariantOverrides {
-    body1: true;
+    body1: true; //commonly "body"
     body2: false;
-    body: true;
     button: false;
     kbd: true;
     legend: true;
     overline: false;
-    subtitle1: true;
+    subtitle1: true; // commonly "caption"
```

Comment:
Until Design/Eng are fully aligned on the available styles and what we call them, I thought these notes might help ease any confusion that arises.

Reply:
They're confusing to me, maybe if we said "// this value is "caption" in the styleguide."